<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerticalLayout</class>
 <widget class="QWidget" name="VerticalLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="title">
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Rubik Light</family>
       <pointsize>34</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 34pt &quot;Rubik Light&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Inserisci i tuoi dati</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="age_label">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Rubik Light&quot;;
background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string>Età</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="age">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Rubik Light&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="gender_label">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Rubik Light&quot;;
background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string>Genere</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="gender">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Rubik Light&quot;;
selection-background-color: rgb(0, 64, 130);</string>
          </property>
          <item>
           <property name="text">
            <string>M</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>F</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="nationality_label">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Rubik Light&quot;;
background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string>Nazionalità</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nationality">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Rubik Light&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="english_level_label">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Rubik Light&quot;;
background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string>Livello di Inglese</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="english_level">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Rubik Light&quot;;
selection-background-color: rgb(0, 64, 130);
selection-color: rgb(255, 255, 255);</string>
          </property>
          <item>
           <property name="text">
            <string>A1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>A2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>C1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>C2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Madrelingua</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="error_text">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Rubik Light&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(255,0,0);</string>
     </property>
     <property name="text">
      <string>Per favore compila tutti i campi.</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="done_button">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
font: 15pt &quot;Rubik Light&quot;;
color: rgb(255,255,255);
background-color: rgb(0, 64, 130);</string>
     </property>
     <property name="text">
      <string>Avanti</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
